### PR TITLE
Added Location and AR quest steps (Expo update)

### PR DESCRIPTION
**Pull request by ponjoru**

### CODE REVIEW

File: src/components/ValidateImage/actions.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+export function imageCompare(body){
+    return{
+        type: IMAGE_COMPARE,
+        body,
+    }
+}
+
+/**
+ * Dispatched when museum data is loaded by the request saga
```
Comment (maik-nack) [inline src/components/ValidateImage/actions.js]: the comment does not match the function

File: src/components/ValidateImage/actions.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+ */
+export function imageCompareFinished(result) {
+    return {
+        type: IMAGE_COMPARE_SUCCESS,
+        result,
+    };
+}
+
+/**
+ * Dispatched when loading museums data fails
```
Comment (maik-nack) [inline src/components/ValidateImage/actions.js]: the comment does not match the function

Comment (ponjoru) [reply src/components/ValidateImage/actions.js]: thank you, fixed already

File: src/containers/Question/CustomCamera.js [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+
+        const screen = Dimensions.get('window');
+
+        const hToCrop = (this.state.targetHeight / screen.height) * H;
+        const wToCrop = (this.state.targetWidth / screen.width) * W;
+
+        let manipResult;
+        try {
+            manipResult = await ImageManipulator.manipulate(
+                image.uri,
```
Comment (maik-nack) [inline src/containers/Question/CustomCamera.js]: it looks like one action with string 84

File: src/containers/Question/QuestMap.js [hunk 4]
```diff
@@ -0,0 +1,266 @@
...
+                this.setState({location:newLocation});
+                this.timestamp = currTime;
+            }
+        }*/
+        console.log('Location change: ', newLocation);
+        this._validateResult();
+    };
+
+    _calcDistance(currCoordinates){
+        const finnish = this.state.borders[1];
```
Comment (maik-nack) [inline src/containers/Question/QuestMap.js]: maybe there is an error

File: src/containers/Question/QuestMap.js [hunk 5]
```diff
@@ -0,0 +1,266 @@
...
+                    longitude: parseFloat(toGoLoc.lon)
+                };
+                this.setState({borders: this.state.borders.concat(initialCoords).concat(coordsToGo), coordinate});
+            }
+
+        this.setState({
+           latitude,
+           longitude,
+           routeCoordinates: routeCoordinates.concat([newCoordinate]),
+           distanceTravelled: distanceTravelled + this._calcDistance(newCoordinate),
```
Comment (maik-nack) [inline src/containers/Question/QuestMap.js]: maybe there is an error

**Pull request merged**